tools/src/main/java/org/apache/kafka/tools/VerifiableProducer.java

```diff
@@ -256,6 +256,12 @@ String shutdownString() {
         return toJsonString(data);
     }
 
+    String startupString() {
```

Comment:
Lol, not sure how we ended up with 2 different ways of generating these outputs in VerifiableProducer vs VerifiableConsumer...